README.rst

```diff
@@ -75,6 +75,8 @@ want to override logging and exception handling):
         def send_message(self, message):
             print('Sending bytes in some custom way: {!r}'.format(message))
 
+If you want to propagate exceptions when messages are sent, set ``raise_send_errors = True`` during initialization
```

Comment:
How about this:

```
By default, exceptions that occur when sending a message are logged. If you
want to raise and propagate exceptions instead, instantiate ``Sender`` with
``raise_send_errors=True``. It's an error to set ``raise_send_errors`` when
``interval`` is specified.
```

Note no space around the `=` for the keyword arg, and the final period / full stop.

Also, for better or worse, this files lines are wrapped. Could we wrap to a similar width as the other lines?